### PR TITLE
rename business calculator to data collection

### PR DESCRIPTION
**Pull request by nanditagoclimate**

### CODE REVIEW

File: app/views/admin/data_collections/_form.html.erb [hunk 1]
```diff
@@ -122,4 +122,4 @@
   <div class="text-right">
     <%= form.button class: 'button button-cta', type: 'button', 'data-disable-with': 'Please wait..', 'data-action': 'data-collection-form#submit' %>
```
Comment (nanditagoclimate) [inline app/views/admin/data_collections/_form.html.erb:123]: On clicking  'create data collection', the form is being created in database, by the app is not going back to listing page. Unable to figure out why. Need @emmabckstrm 's help here please

File: yarn.lock [hunk 2]
```diff
@@ -2079,9 +2079,9 @@ caniuse-api@^3.0.0:
     lodash.uniq "^4.5.0"
 
 caniuse-lite@^1.0.0, caniuse-lite@^1.0.30001165, caniuse-lite@^1.0.30001181, caniuse-lite@^1.0.30001196:
-  version "1.0.30001258"
-  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001258.tgz"
-  integrity sha512-RBByOG6xWXUp0CR2/WU2amXz3stjKpSl5J1xU49F1n2OxD//uBZO4wCKUiG+QMGf7CHGfDDcqoKriomoGVxTeA==
+  version "1.0.30001280"
+  resolved "https://registry.npmjs.org/caniuse-lite/-/caniuse-lite-1.0.30001280.tgz"
+  integrity sha512-kFXwYvHe5rix25uwueBxC569o53J6TpnGu0BEEn+6Lhl2vsnAumRFWEBhDft1fwyo6m1r4i+RqA4+163FpeFcA==
```
Comment (stefanlindbohm) [inline yarn.lock:2084]: If I'm gonna be super picky, this is unrelated to the changes of this PR and should be done separately.

**Review by stefanlindbohm — APPROVED** [review]
Good work tracking all usage points down!

I'm adding mostly comments that are down to taste so I'll let you decide what to pick up on and what to leave as is.

File: app/models/data_collections/form.rb [hunk 3]
```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+module DataCollections
```
[the remark this answers is not in the document]
Comment (emmabckstrm) [reply app/models/data_collections/form.rb:3]: I agree

**Review by emmabckstrm — CHANGES_REQUESTED** [review]
Really, really great work! I agree with Stefan's point on pluralization of `DataCollection`. 

I did also test the branch locally and I was not able to save a DataCollection::Form from Admin view, I tried with an existing calculator that was in draft status. See error below.
Screenshot 2021-11-22 at 11 07 15

I also tried to create a new calculator, which worked fine, but I can't seem to add any questions to my sections, and I don't get any apparent errors as a user. So there seem to be something fishy with the questions. Other than that I seem to be able to use the form as usual :blush:

Small thing: In the admin menu, the Data Collection Form item is not highlighted when I have clicked on it. This is fixed in file `app/views/layouts/admin.html.erb` on line 65, where it's looking for controllers with name `business_calculators`.

**Pull request merged**